app.py

```diff
@@ -11,8 +15,8 @@ def home_page():
 @app.route('/', methods=['POST', 'GET'])
```

Comment:
I think the only method used here is `POST`. Even more, the default methods is probably `GET` so you have defined it twice :)